src/API/Google/AdsCampaign.php

```diff
@@ -27,6 +27,7 @@
 use Google\Ads\GoogleAds\V20\Services\GoogleAdsRow;
 use Google\Ads\GoogleAds\V20\Services\MutateGoogleAdsRequest;
 use Google\Ads\GoogleAds\V20\Services\MutateOperation;
+use Google\Ads\GoogleAds\V21\Enums\EuPoliticalAdvertisingStatusEnum\EuPoliticalAdvertisingStatus;
```

Comment:
Should be V20 not V21?

Move it after `use Google\Ads\GoogleAds\V20\Enums\AdvertisingChannelTypeEnum\AdvertisingChannelType;`

Reply:
We don't have it in v20 ([ref](https://github.com/googleads/google-ads-php/tree/main/metadata/Google/Ads/GoogleAds/V20/Enums)), its available in v21 only ([ref](https://github.com/googleads/google-ads-php/tree/main/metadata/Google/Ads/GoogleAds/V21/Enums))

Reply:
Here it is https://github.com/googleads/google-ads-php/blob/df36b22ad1ca82e8f931d6c171f0e5ef6e894587/src/Google/Ads/GoogleAds/V20/Enums/EuPoliticalAdvertisingStatusEnum/EuPoliticalAdvertisingStatus.php

Reply:
There is a [post-script](https://github.com/woocommerce/google-listings-and-ads/blob/3.3.1/composer.json#L62) after running composer install, which removes unused Google Ads library files to reduce the plugin's bundle size. Therefore, when importing a new class that has not been used before from the `Google\Ads\GoogleAds` namespace, it needs to fully remove the `vendor` directory and reinstall composer dependencies.

Reply:
👀 https://github.com/googleapis/googleapis/blob/b1e61e0ad25d5e6050e8da295f5e14f999f06632/google/ads/googleads/v20/enums/eu_political_advertising_status.proto#L35-L49